Source/DafnyCore/AST/Grammar/Printer.cs

```diff
@@ -1018,6 +1018,9 @@ public class Printer {
 
       int ind = indent + IndentAmount;
       PrintSpec("requires", method.Req, ind);
+      if (method.Reads.Expressions != null) {
```

Comment:
An annoying oversight on my part in the original PR. FYI I'd really like to switch the library backend on for `%testDafnyForEachCompiler` so that we get more automated testing of the pretty printer (since .doo files round trip programs through it)